acquia/blt
# Fixes #3206 to include sensiolabs/security-checker and new composer s…

…ecurity test.

Fixes #3206 
--------

Changes proposed:
---------
(What are you proposing we change?)

- adds sensiolabs/security-checker to composer.json
- adds new tests:security:check:composer command
- bundles tests:security:check:composer into blt tests:all
- updates docs to reference new command

Steps to verify the solution:
----------
(If this PR is not fixing a defect/bug, you can remove this section.)

1. update blt to include this patch
2. composer install (and ensure that sensiolabs/security-checker is installed)
2. run blt tests:security:check:composer

Outstanding question:
- how to make sure that the composer.json file is updated for people already updated to 10.0.0? Do we need to run a new update script? @grasmash / @danepowell  how has this been handled in the past?
